pyglmnet/pyglmnet.py

```diff
@@ -446,25 +446,56 @@ def score(self, y, yhat, ynull=None, method='deviance'):
         method : str
             One of 'pseudo_R2' or 'deviance'
         """
-        y = y.ravel()
-        if self.distr != 'multinomial':
-            yhat = yhat.ravel()
 
-        L1 = utils.log_likelihood(y, yhat, self.distr)
-        if self.distr in ['poisson', 'poissonexp']:
-            LS = utils.log_likelihood(y, y, self.distr)
-        else:
-            LS = 0
+        if (self.distr != 'multinomial' and len(yhat.shape) == 1) or \
+                (self.distr == 'multinomial' and len(yhat.shape) == 2):
```

Comment:
why do we need to check both `self.distr == 'multinomial'`and `len(yhat.shape) == 2` ? shouldn't one of them be enough?